Sources/xcodeproj/PBXTarget.swift

```diff
@@ -138,8 +138,13 @@ public extension PBXTarget {
     /// - Throws: an error if the build phase cannot be obtained.
     public func sourcesBuildPhase() throws -> PBXSourcesBuildPhase? {
         return try buildPhasesReferences
-            .compactMap({ try $0.object() as PBXSourcesBuildPhase })
+
+            // Cannot cast to PBXSourcesBuildPhase here.
```

Comment:
Good catch @toshi0383. I'd remove the comment and then it's ready to merge.

Reply:
Removed this line and left second line as is. (Is that what you mean?)
557d6a4